snippets/csharp/VS_Snippets_CLR/conceptual.interop.marshaling/cs/msgbox.cs

```diff
@@ -9,20 +9,20 @@ public class LibWrap
     [DllImport("User32.dll", EntryPoint = "MessageBox",
         CharSet = CharSet.Auto)]
     public static extern int MsgBox(
-        int hWnd, string text, string caption, uint type);
+        IntPtr hWnd, string text, string caption, uint type);
```

Comment:
Should the parameters be renamed to `lpText`, `lpCaption` and `uType` to follow the naming conventions (as I did in dotnet/docs#11450)?

In https://docs.microsoft.com/dotnet/standard/native-interop/best-practices

> ✔️ DO use the same naming and capitalization for your methods and parameters as the native method you want to call.

Reply:
Yes, that would be better. Thanks.